tests/Fixer/FunctionNotation/FunctionDeclarationFixerTest.php

```diff
@@ -22,14 +22,31 @@
  */
 final class FunctionDeclarationFixerTest extends AbstractFixerTestCase
 {
+    private static $configurationClosureSpacingNone = array('closure_function_spacing' => 'none');
+
+    public function testInvalidConfigurationClosureFunctionSpacing()
+    {
+        $this->setExpectedExceptionRegExp(
+            'PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException',
+            '#^\[function_declaration\] Spacing is invalid. Should be one of: "none", "one".$#'
+        );
+
+        $this->fixer->configure(array('closure_function_spacing' => 'neither'));
+    }
+
     /**
      * @param string      $expected
      * @param null|string $input
+     * @param null|array  $configuration
      *
      * @dataProvider provideCases
      */
-    public function testFix($expected, $input = null)
+    public function testFix($expected, $input = null, $configuration = null)
```

Comment:
everywhere - please add typehinting for new parameter, it's array, so `array $configuration = null`.